app/views/users/index.html.erb

```diff
@@ -1 +1,11 @@
-<h1>User Details</h1>
\ No newline at end of file
+<ul>
+  <% if @users.present? %>
+    <% @users.each do |user| %>
+      <li>
+        <%= render 'partials/user', user: user %>
```

Comment:
- [ ] Generally your design does not match the given wireframe and this implies to all your views pages. kindly look at the [project requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md) and get your design to be close to the given wireframe